jest-puppeteer.config.js

```diff
@@ -1,3 +1,5 @@
+const { mapSeries } = require("async");
```

Comment:
Any idea why this include is being added here? It doesn't seem to do anything...